package.json

```diff
@@ -29,7 +29,7 @@
     "@oclif/config": "^1.17.0",
     "@salesforce/command": "^3.1.0",
     "@salesforce/core": "^2.18.0",
-    "@salesforce/lwc-dev-mobile-core": "^1.0.0-beta9",
+    "@salesforce/lwc-dev-mobile-core": "1.0.0-beta10",
```

Comment:
should this be `^1.0.0-beta10` instead?

Reply:
I moved `^` then on CI it kept pulling `1.0.0-beta9` instead and would fail the build. `^` supposed to mean take the latest minor or patch so I thought the one you published earlier would be picked up. Unless I can find why the `^` is not working I want to not use `^`.